views/index.jade

```diff
@@ -3,16 +3,22 @@ extends layout
 block content
     div.row
 
-        // KEEP CONTENT TO THE LEFT
         // DISPLAY LOGIN OR REGISTER WHEN NOT LOGGED IN
         if (!currentUser)
-            //p.tip-add-message Login or register to add a tip
-        div.col-sm-6.col-sm-offset-3.tip-form
+            //p.itinerary-add-message Login or register
+        div.col-sm-6.col-sm-offset-3.itinerary-form
             // DISPLAY FORM ONLY WHEN USER IS LOGGED IN
             //- if (currentUser)
             .list-group
                 each itinerary, i in itineraries
-                    a.list-group-item(href='/itineraries/#{itinerary.id}/')
-                        h4.list-group-item-heading #{itinerary.name}
-                        p.list-group-item-text #{itinerary.date}
-                        p.list-group-item-text #{itinerary.description}
+                   .itinerary-list-container
+                      .itinerary-list-container-body
+                        a.list-group-item(href='/itineraries/#{itinerary.id}/')
+                            h4.list-group-item-heading #{itinerary.name}
+                            p.list-group-item-text #{itinerary.date}
+                            p.list-group-item-text #{itinerary.description}
+                        form.itinerary-delete
```

Comment:
After loading the page, the delete buttons do not look like they are in the right place. Please double check
![screen shot 2018-01-15 at 10 48 39 pm](https://user-images.githubusercontent.com/8893980/34975341-4509d4e4-fa46-11e7-9652-19b6d0bf2fd2.png)